qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -542,15 +542,101 @@ def from_operator(obj, atol=None, rtol=None):
 
     @staticmethod
     def from_list(obj):
-        """Construct from a list [(pauli_str, coeffs)]"""
+        """Construct from a list of Pauli strings and coefficients.
+
+        For example, the 5-qubit Hamiltonian
+
+        .. math::
+
+            H = X_0 Z_3 + 2 Y_1 Y_4
+
+        can be constructed as
+
+        .. code-block:: python
+
+            # via tuples and the full Pauli string
+            op = SparsePauliOp.from_list([("XIIZI", 1), ("IYIIY", 2)])
+
+        Args:
+            obj (Iterable[Tuple[str, complex]]): The list of 2-tuples specifying the Pauli terms.
+
+        Returns:
+            SparsePauliOp: The SparsePauliOp representation of the Pauli terms.
+
+        Raises:
+            QiskitError: If the list of Paulis is empty.
+        """
         obj = list(obj)  # To convert zip or other iterable
+
+        size = len(obj)  # number of Pauli terms
+        if size == 0:
+            raise QiskitError("Input Pauli list is empty.")
+
+        # determine the number of qubits
         num_qubits = len(obj[0][0])
-        size = len(obj)
+
         coeffs = np.zeros(size, dtype=complex)
         labels = np.zeros(size, dtype=f"<U{num_qubits}")
         for i, item in enumerate(obj):
             labels[i] = item[0]
             coeffs[i] = item[1]
+
+        paulis = PauliList(labels)
+        return SparsePauliOp(paulis, coeffs, copy=False)
+
+    @staticmethod
+    def from_sparse_list(obj, num_qubits):
+        """Construct from a list of local Pauli strings and coefficients.
+
+        Each list element is a 3-tuple of a local Pauli string, indices where to apply it,
+        and a coefficient.
+
+        For example, the 5-qubit Hamiltonian
+
+        .. math::
+
+            H = X_0 Z_3 + 2 Y_1 Y_4
+
+        can be constructed as
+
+        .. code-block:: python
+
+            # via triples and local Paulis with indices
+            op = SparsePauliOp.from_list([("XZ", [0, 3], 1), ("YY", [1, 4], 2)], num_qubits=5)
```

Comment:
The example here is correct, but the function itself has the wrong endianness:
```python
In [3]: from qiskit.quantum_info import SparsePauliOp
   ...: SparsePauliOp.from_sparse_list([("XZ", [0, 3], 1)], 5).paulis[0][0]
Out[3]: Pauli('I')
```
(it should be `Pauli('X')`).